build.py

```diff
@@ -111,7 +111,7 @@ def step_set_feature_file(path: Path, name: str, instance: ufoLib2.Font) -> None
     for item in featureList:
         if "PL" in name and item == "rclt":
             featureSet += Path(path / str("rclt_PL.fea")).read_text()
-        if "NF" in name and item == "rclt":
+        elif "NF" in name and item == "rclt":
```

Comment:
This may have caused issues with the later `else`, resulting in the inclusion of `rclt.fea` after we'd already included `rclt_PL.fea` in the PL variant.